deploy/requirements.yml

```diff
@@ -41,4 +41,4 @@
 
 - name: opencraft
   src: https://github.com/open-craft/ansible-opencraft
-  version: 'origin/smarnach/opencraft'
+  version: a3e9cde2968e9de94cb9e908244d05654def8dcd
```

Comment:
@jbzdak yes, but than it might make sense to actually do versioning, i.e. using git tags. Does it make sense?